pgcli/pgclirc

```diff
@@ -131,6 +131,9 @@ multiline_continuation_char = ''
 # The string used in place of a null value.
 null_string = '<null>'
 
+# manage pager on startup
+pset-pager = on
```

Comment:
Let's change the name to `enable_pager` and use the values `True` and `False`. 

Let's move this option close to the `pager` option in the config file (line 76).